### ISSUE TITLE
Setting the HTML_DYNAMIC_SECTION to NO has no effect (Origin: bugzilla #777462)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#6149]
status NEEDINFO severity *normal* in component *general* for *---*
Reported in version *1.8.13* on platform *Other*
Assigned to: Dimitri van Heesch

On 2017-01-18 18:57:45 +0000, Corentin SÃ©chet wrote:

> Setting the HTML_DYNAMIC_SECTION option to NO in a Doxyfile will still generate a dynamic javascript menu in the output. It's annoying as some themes rely on a static menu.
> 
> The bug is present in Debian sid (Doxygen 1.8.13), as well as in the windows build at the same version, from the installer downloadable from the Doxygen website.

On 2017-01-19 19:16:12 +0000, albert wrote:

> Are you mixing the HTML_DYNAMIC_SECTION option with the new option HTML_DYNAMIC_MENUS option which was introduced after the release of doxygen 1.8.13?

**Comment by iteco** [comment]
Reproduced
I was trying to get around of the issue #6143 (see image and version numbers from there) by applying doxygen conf setting 
`HTML_DYNAMIC_MENUS = NO`
but it didn't helped. I didn't got the menu visible.

**Comment by albert-github** [comment]
@iteco is this problem that you have with `HTML_DYNAMIC_MENUS = NO` a security issue in relation to Jenkins and doxygen 1.8.17 as you referenced #6143 or can you reproduce the problem locally as well
- What happens when you use locally the newest version of doxygen and `HTML_DYNAMIC_MENUS = NO`
- What happens when you use locally the newest version of doxygen and `HTML_DYNAMIC_MENUS = YES`
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem locally? Please don't add external links as they might not be persistent.

**Comment by iteco** [comment]
@albert-github 
yes, they are related in a way that the same system was used in both setups. I tried to generate report with doxygen, but I didn't get a site/page with content/navigation. I tried to solve it first by making doxygen to generate static navigation. It didn't worked, as reported in this issue. 

After that I looked if I would get it working some other way, and I found a way by relaxing security settings too much as reported in that #6143.

I don't have a local setup, and unfortunately my computer is now in a state that I cannot try that out. However, I have that setup from which I made the report still on the remote machine. I can make a zip file from that directory if it helps? 
(so that the report is generated with both of the settings `HTML_DYNAMIC_MENUS = NO` and `HTML_DYNAMIC_MENUS = YES.`)

**Comment by albert-github** [comment]
When the zip file is not overly large it would be an option, as always a normal small, self contained example would be the better choice.
Also in the question and comment there is a mentioning of the doxygen version 1.8.13 and 1.8.17, did / can you test with the current 1.9.1 version as well?

**Comment by iteco** [comment]
Thanks @albert-github for the quick answer. Unfortunately I could not do the same. 

However, today I found time to check this again and I planned to produce those files, but something had changed and it is now working. I haven't changed anything (at least so that I would know), so I suspect that the problem has been in cached web page in my browser or in Cloudflare. 

Anyway it works for me now with settings
HTML_DYNAMIC_MENUS     = NO
HTML_DYNAMIC_SECTIONS  = NO

**Comment by albert-github** [comment]
Seen the last comment the problem is not reproducible anymore / fixed.
Furthermore quite a few things changed so I think we can say solved in the mean time and / or usage
